Sources/GDSCommon/Patterns/GDSInformation/GDSInformationViewModel.swift

```diff
@@ -32,3 +32,8 @@ public protocol GDSInformationViewModel {
     var primaryButtonViewModel: ButtonViewModel { get }
     var secondaryButtonViewModel: ButtonViewModel? { get }
 }
+
+@MainActor
+public protocol GDSInformationViewModelWithChildView {
```

Comment:
Does this also need to conform to GDSInformationViewModel?

Reply:
@Oliver-Binns not sure what is best practice here, do we conform to both of these in the new protocol or do we conform to both in the new view model that we create?

Reply:
I wasn't sure on which way was best here either. In other areas it looks like we're conforming the view model to both protocols, but happy to change this new protocol to conform if that's preferred

Reply:
It stems from [ISP](https://en.wikipedia.org/wiki/Interface_segregation_principle) really. I'd be tempted to keep them separate. 

If we want to make life easier for consumers ensuring they only have to conform to _one_ thing we could do something like:
`GDSInformationViewModel` -> main protocol for the view
`GDSViewModelWithChildView` -> protocol with child view

then have a typealias to join them:
`typealias GDSInformationViewModelWithChildView = GDSInformationViewModel & GDSViewModelWithChildView` but that might be overkill?

Reply:
We do have a `typealias` for `GDSErrorViewModel` so there's definitely precedent in the code for it, but if we're happy as is and all are happy knowing to conform to both protocols (and we feel it's explained sufficiently) then why complicate things?